selfdrive/ui/ui.h

```diff
@@ -60,6 +60,7 @@ typedef struct UIScene {
   cereal::PandaState::PandaType pandaType;
 
   cereal::LongitudinalPersonality personality;
+  cereal::LongitudinalPlanSP::AccelerationPersonality accel_personality;
```

Comment:
Where is this used? If it's not being used in this PR yet, let's either leave a `TODO-SP` with what it may be used, or remove it and reimplement in a later PR.